Enables itself even when --disable-backtrace is set,
and doesn't build on musl or ppc* (not tested).

diff --git Cargo.toml Cargo.toml
index 4076a60577..945f26f638 100644
--- Cargo.toml
+++ Cargo.toml
@@ -13,7 +13,6 @@ members = [
   "netwerk/test/http3server",
   "security/manager/ssl/osclientcerts",
   "testing/geckodriver",
-  "toolkit/crashreporter/rust_minidump_writer_linux",
   "toolkit/library/gtest/rust",
   "toolkit/library/rust/",
   "toolkit/mozapps/defaultagent/rust",
@@ -79,7 +78,6 @@ spirv_cross = { git = "https://github.com/kvark/spirv_cross", branch = "wgpu4" }
 # failure's backtrace feature might break our builds, see bug 1608157.
 failure = { git = "https://github.com/badboy/failure", rev = "64af847bc5fdcb6d2438bec8a6030812a80519a5" }
 failure_derive = { git = "https://github.com/badboy/failure", rev = "64af847bc5fdcb6d2438bec8a6030812a80519a5" }
-minidump_writer_linux = { git = "https://github.com/msirringhaus/minidump_writer_linux.git", rev = "9191af36343846b2c7ada65b9602b481b717c4d8" }
 
 [patch.crates-io.cranelift-codegen]
 git = "https://github.com/mozilla-spidermonkey/wasmtime"
diff --git toolkit/moz.configure toolkit/moz.configure
index 806576b1ef..43ef8309f1 100644
--- toolkit/moz.configure
+++ toolkit/moz.configure
@@ -2487,8 +2487,6 @@ set_define("MOZ_USE_NATIVE_POPUP_WINDOWS", True, when="MOZ_USE_NATIVE_POPUP_WIND
 # ==============================================================
 @depends(target)
 def oxidized_breakpad(target):
-    if target.kernel == "Linux" and target.os != "Android":
-        return target.cpu in ("x86", "x86_64")
     return False
